.deploy/nais/app-preprod.yaml

```diff
@@ -50,13 +51,30 @@ spec:
         - application: familie-prosessering
           namespace: teamfamilie
           cluster: dev-gcp
+      external:
+        - host: kafka-schema-registry.nais-q.adeo.no
+        - host: pdl-api.dev-fss-pub.nais.io
+        - host: familie-integrasjoner.dev-fss-pub.nais.io
```

Comment:
Burde man ha tatt med de andre appene som outband rules? De var jo ikke i bruk per dato, men kanskje de skrur de på? Og hvorfor bruker man external mot familie-integrasjoner?
external:
   rules:
        - application: familie-ba.sak
          cluster: dev-gcp

Reply:
ja, måtte inn noen regler i firewall for å åpne for trafikk fra gcp inn mot on-prem, skal sjekke om regelen er knyttet til app eller generell

Reply:
ser ut til at fw regel er generell -> https://github.com/navikt/bigip-iac/blob/main/pub-fss/records/prod.yaml